### PR TITLE
Add test for RamClustR(CSV) wrapper

### PR DESCRIPTION
**Pull request by maximskorik**

### CODE REVIEW

File: tools/ramclustr/test-data/test-spectra_csv.msp [hunk 1]
```diff
@@ -0,0 +1,357 @@
+NAME:C01
```
Comment (martenson) [inline tools/ramclustr/test-data/test-spectra_csv.msp:1]: Why is this file here?

Comment (maximskorik) [reply tools/ramclustr/test-data/test-spectra_csv.msp:1]: @martenson, thanks for the note. Forgot to write that output to the wrapper, fixed that in the last commit.

**Review by martenson — APPROVED** [review]
